core/src/main/java/io/grpc/internal/Channelz.java

```diff
@@ -33,7 +33,7 @@
       new ConcurrentHashMap<Long, Instrumented<ServerStats>>();
   private final ConcurrentMap<Long, Instrumented<ChannelStats>> rootChannels =
```

Comment:
fyi, this will need to be scannable so CHM will not be a long term solution.

Reply:
I realized this when implementing the channelz service, but I wanted to keep this PR focused on one change.